src/routes/(dashboard)/notifications-helpers.ts

```diff
@@ -12,20 +13,22 @@ export const notificationsContentColumns: column<FlattenedNotificationsContent>[
 export const flattenNotificationsContent = (
     notificationsContent: NotificationsContent[]
 ): FlattenedNotificationsContent[] => {
-    return notificationsContent.map((notification) => {
-        return {
-            id: notification.comment?.id,
-            name: notification.comment?.user.name,
-            time: formatNotificationsDateString(notification.comment?.created ?? ''),
-            notification: notification.comment?.text,
-            notoficationId: notification.comment?.id,
-            isRead: notification.isRead,
-            resourceContentId: notification.comment?.resourceContentId,
-            parentResourceDisplayName: notification.comment?.parentResourceDisplayName,
-            title: notification.comment?.resourceEnglishLabel,
-            kind: notification.kind,
-        };
-    });
+    return notificationsContent
+        .filter((notification) => notification.kind === NotificationKind.comment)
```

Comment:
Note that you can do this filtering on the server side: https://github.com/BiblioNexusStudio/aquifer-server/blob/3e596877659d4624d688125c034450fcfdbcea24/src/Aquifer.API/Endpoints/Notifications/List/Request.cs#L8